monitoring/prometheus/main.py

```diff
@@ -0,0 +1,71 @@
+"""
```

Comment:
Can you remove the triple quotations around the license header here and there?

Reply:
Done, thank you.